doc/HTTP Consumer API.md

```diff
@@ -0,0 +1,120 @@
+# Consumer API

```

Comment:
Can you rename this file using underscores and not spaces please? The same for the HTTP producer API doc.